catalog/app/website/pages/OpenLanding/Contribute/Contribute.js

```diff
@@ -122,18 +122,18 @@ export default function Contribute() {
           </M.Typography>
           <M.Box pt={4} />
           <M.Typography variant="body1" color="textSecondary">
-            Quilt is evolving Join our{' '}
+            Quilt is evolving. Join the{' '}
+            <M.Link color="secondary" underline="always" href="http://eepurl.com/bOyxRz">
+              <b>general mailing list</b>
+            </M.Link>{' '}
+            {' or '}
             <M.Link
               href="https://groups.google.com/forum/#!forum/quilt-data-dev/"
               color="secondary"
               underline="always"
             >
-              <b>developer mailing list</b>
+              <b>and developer forums</b>
             </M.Link>
```

Comment:
i think you need to add space here btw the link and the text that follows
```suggestion
             </M.Link>{' '}
```